tests/unit/test_peak_collection.py

```diff
@@ -5,13 +5,34 @@
 import pytest
 
 
-def test_peak_shape_enum():
-    assert PeakShape.getShape('gaussian') == PeakShape.GAUSSIAN
-    assert PeakShape.getShape('GAUSSIAN') == PeakShape.GAUSSIAN
+def check_peak_shape_enum(peak_shape, num_native_params):
```

Comment:
The parameterization of the test is a good idea. Lines 8-25 could be converted into `@pytest.mark.parametrize` and possibly be clearer. I don't think it is necessary though unless we expand to test more peak shapes.